### DISCUSSION TITLE
`Bun` support

### DISCUSSION THREAD

**Discussion by RoyRao2333** [web-infra-dev/rsbuild#4005] [General]
Does `rsbuild` supports `Bun` in addition to `Node.js`?

**Comment by chenjiahan** [comment]
Yes, Rsbuild supports running in Bun or Deno, you can try "bun run --bun build"

**Reply by RoyRao2333** [reply]
Great!

I was using rsbuild + Node before, and now I wanna try Bun in my new project, so I'm here to ask.

Thanks for the reply 😊
